service/tests/Core.FunctionalTests/Core.FunctionalTests.csproj

```diff
@@ -14,10 +14,8 @@
     </PropertyGroup>
 
     <ItemGroup>
```

Comment:
No need to add project references here, I removed also old leftovers

Reply:
Ok you are right.